### ISSUE TITLE
Reference proteome similarity error

### ISSUE THREAD

**Issue by rbowenj** [griffithlab/pVACtools#723]
Hi,

I'm running pVACseq version 2.0.4 via Docker. 
When I use the `--run-reference-proteome-similarity` flag I get the error below. I also encountered this error when using pVACtools version 2.0.3, but not version 2.0.2.

Error log:
```2021-10-19T13:09:31.812677510Z WARNING:root:Transcript WildtypeProtein sequence contains internal stop codon. These can occur in Ensembl transcripts of the biotype polymorphic_pseudogene. Skipping.
2021-10-19T13:09:31.812709187Z 6 29385895 C CAGCGA ENST00000514827
2021-10-19T13:48:04.895155243Z /usr/local/lib/python3.7/site-packages/lib/aggregate_all_epitopes.py:175: FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only
2021-10-19T13:48:04.895243935Z   df.drop('rank_tier', 1, inplace=True)
2021-10-19T13:48:04.895246905Z /usr/local/lib/python3.7/site-packages/lib/aggregate_all_epitopes.py:176: FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only
2021-10-19T13:48:04.895250068Z   df.drop('rank_ic50', 1, inplace=True)
2021-10-19T13:48:04.895252818Z /usr/local/lib/python3.7/site-packages/lib/aggregate_all_epitopes.py:177: FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only
2021-10-19T13:48:04.895256222Z   df.drop('expr', 1, inplace=True)
2021-10-19T13:48:04.895258992Z /usr/local/lib/python3.7/site-packages/lib/aggregate_all_epitopes.py:178: FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only
2021-10-19T13:48:04.895262149Z   df.drop('rank_expr', 1, inplace=True)
2021-10-19T13:48:04.895264910Z /usr/local/lib/python3.7/site-packages/lib/aggregate_all_epitopes.py:179: FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only
2021-10-19T13:48:04.895268129Z   df.drop('rank', 1, inplace=True)
2021-10-19T13:48:04.903088912Z Traceback (most recent call last):
2021-10-19T13:48:04.903113034Z   File "/usr/local/bin/pvacseq", line 8, in <module>
2021-10-19T13:48:04.903118553Z     sys.exit(main())
2021-10-19T13:48:04.903122952Z   File "/usr/local/lib/python3.7/site-packages/tools/pvacseq/main.py", line 95, in main
2021-10-19T13:48:04.903128173Z     args[0].func.main(args[1])
2021-10-19T13:48:04.903132736Z   File "/usr/local/lib/python3.7/site-packages/tools/pvacseq/run.py", line 122, in main
2021-10-19T13:48:04.903137556Z     pipeline.execute()
2021-10-19T13:48:04.903141676Z   File "/usr/local/lib/python3.7/site-packages/lib/pipeline.py", line 504, in execute
2021-10-19T13:48:04.903146610Z     PostProcessor(**post_processing_params).execute()
2021-10-19T13:48:04.903151230Z   File "/usr/local/lib/python3.7/site-packages/lib/post_processor.py", line 37, in execute
2021-10-19T13:48:04.903155888Z     self.calculate_reference_proteome_similarity()
2021-10-19T13:48:04.903160228Z   File "/usr/local/lib/python3.7/site-packages/lib/post_processor.py", line 139, in calculate_reference_proteome_similarity
2021-10-19T13:48:04.903175979Z     file_type=self.file_type
2021-10-19T13:48:04.903180667Z   File "/usr/local/lib/python3.7/site-packages/lib/calculate_reference_proteome_similarity.py", line 140, in execute
2021-10-19T13:48:04.903185611Z     peptide = self.extract_n_mer(mt_records_dict[line['Index']], int(line['Sub-peptide Position']), int(line['Mutation Position']), len(mt_amino_acids))
2021-10-19T13:48:04.903190507Z KeyError: '146.CBX4.ENST00000269397.missense.158K/N'```

**Comment by duolin1986** [comment]
same with me

**Comment by susannasiebert** [comment]
@rbowenj @duolin1986 I apologize for having let this issue fall by the wayside. I was able to reproduce this issue in 2.0.4, however it appears to be fixed in 3.1.1. I suggest upgrading pVACtools to the latest version.
